js/test/test.js

```diff
@@ -327,27 +368,26 @@ async function runPrivateTests(exchange, symbol) {
 
     await test ('signIn', exchange);
 
-    const balance = await test ('fetchBalance', exchange);
-
+    await test ('fetchBalance', exchange);
     await test ('fetchAccounts', exchange);
```

Comment:
@ttodua I think we should find a way to properly handle an error here. Let's say `fetchAccounts` fails for some reason I think that should not prevent the other tests from running, but we want to fail anyway, of course. 

What if `test` returned the error if any, and in the end we verified if any of the tests had failed if so we threw an exception so the process could terminate with an error code. 

Rough example:

```Javascript
    let tests = [
        'fetchBalance',
        'fetchAccounts',
        'fetchOpenOrders',
        'fetchMyTrades',
    ]
    let errors = [];
    for (let i = 0; i < tests.length; i++) {
        const test = tests[i];
        const error = await test (test, exchange, symbol);
       if (error) {
           errors.push(error);
       }
    }

    if (errors.length > 0) {
        throw new Error(errors.join(', '));
    }
```
What do u think?

Reply:
Carlos, ofc it might be perfect for a separate PR, but again, I try to have minimal changes (what you offer, is quite structural change) and even with simpler changes I have to wait months, and please, lets leave that idea for a separate PR, is that ok? I'll take that PR too, but after we are done with existing PRs